packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardTypes.ts

```diff
@@ -0,0 +1,50 @@
+export type ReleaseCardStat = {
+  label: string
+  value: string
+}
+
+export type StatConfig = {
+  // specifies which column should be used to populate the statistic value
+  columnName: string
+  // the label that will be displayed with the statistic in the card
+  label: string
+}
```

Comment:
In the GENIE portal, cards include specimen and patient counts. However, if these cards are reused for other Portal data releases, we may want to show different statistics. Use this config to allow different statistics to be defined for each set of release cards.